<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloneRepositoryDlg</class>
 <widget class="QDialog" name="CloneRepositoryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clone repository</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>From</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="LineEdit" name="txtUrl">
        <property name="mandatory" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="LineEdit" name="txtRemoteName">
        <property name="text">
         <string>origin</string>
        </property>
        <property name="mandatory" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Url</string>
        </property>
        <property name="buddy">
         <cstring>txtUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Remote</string>
        </property>
        <property name="buddy">
         <cstring>txtRemoteName</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>To</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Path</string>
        </property>
        <property name="buddy">
         <cstring>txtPath</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEdit" name="txtPath">
        <property name="mandatory" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnBrowse">
        <property name="text">
         <string>&amp;Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="chkInitSubmodules">
        <property name="text">
         <string>Initialize submodules</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBox" name="chkSubmodulesRecursive">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Recursively</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="txtCheckoutBranch">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Branch to checkout (default if empty)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="chkFetchOne">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Fetch only the branch to be checked out</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QRadioButton" name="chkCloneBare">
        <property name="text">
         <string>Clone Bare Repository</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QRadioButton" name="chkCloneMirror">
        <property name="text">
         <string>Clone as mirror</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="chkCheckout">
        <property name="text">
         <string>Checkout</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>libMacGitverCore/Widgets/LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtUrl</tabstop>
  <tabstop>txtRemoteName</tabstop>
  <tabstop>txtPath</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>chkCloneBare</tabstop>
  <tabstop>chkCloneMirror</tabstop>
  <tabstop>chkCheckout</tabstop>
  <tabstop>txtCheckoutBranch</tabstop>
  <tabstop>chkFetchOne</tabstop>
  <tabstop>chkInitSubmodules</tabstop>
  <tabstop>chkSubmodulesRecursive</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CloneRepositoryDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CloneRepositoryDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
